tests/virt_autotest/install_package.pm

```diff
@@ -35,10 +36,18 @@ sub install_package {
     elsif ($repo_0_to_install =~ /SLE-12-SP1/m) {
         $dependency_repo = 'http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/';
     }
+    elsif ($repo_0_to_install =~ /SLE-15-Installer/m) {
+	$dependency_repo = 'http://download.suse.de/ibs/SUSE:/SLE-15:/GA/standard/'
+    }
     if ($dependency_repo) {
         assert_script_run("zypper --non-interactive --no-gpg-check -n ar -f ${dependency_repo} dependency_repo");
         assert_script_run("zypper --non-interactive --gpg-auto-import-keys ref", 180);
-        assert_script_run("zypper --non-interactive -n in xmlstarlet");
+        if ($repo_0_to_install =~ /SLE-15-Installer/m) {
+        	assert_script_run("zypper --non-interactive -n in bridge-utils");
+        }
+        else {
+		assert_script_run("zypper --non-interactive -n in xmlstarlet");
+	}
```

Comment:
It is better to handle the special rpms for different versions with a variable like $dependent_rpms, just like $dependency_repo, and then use unique command "zypper --non-interactive -n in $dependent_rpms"

Reply:
Updated